ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -171,7 +171,9 @@ module('Acceptance | aws | configuration', function (hooks) {
       await runCmd(`delete sys/mounts/${path}`);
     });
 
-    test('it should not show identityTokenTtl or maxRetries if they have not been set', async function (assert) {
+    test('it should show identityTokenTtl or maxRetries even if they have not been set', async function (assert) {
+      // documenting the intention that we show fields that have not been set but are returned by the api due to defaults
+      // this test also documents that maxRetries returns 0 while the API docs indicate -1 is the default value
```

Comment:
are the docs going to be updated? if so this line may be a little confusing in the future. the assertion message below is clear enough to me.

Reply:
If I get word from the backend this is the intended consequence then I'll update the docs and remove the line. If they say oops then I'll have to update this test if they put in a fix. This comment should be short-lived but I wanted to include a comment in case this issue is long-lived and someone wonders why the docs say -1 but the api is returning 0.